src/main/java/bio/terra/cli/command/workspace/RemoveUser.java

```diff
@@ -21,18 +19,10 @@
       description = "Role to grant: ${COMPLETION-CANDIDATES}")
   private IamRole role;
 
-  @CommandLine.Mixin Format formatOption;
```

Comment:
+1 on this question

Reply:
I thought it was confusing what this returned. Maybe I've just been staring at this too long though. Suggestions welcome.

`remove-user` actually removes a user email + role combination, not an entire user from a workspace. So I could see 3 options for return values here:
- Return the email + role that got removed. That would be consistent with other delete commands that return the deleted thing. But that means we're using `UFWorkspaceUser` to mean different things for different commands. For listing workspace users, it holds email + all roles that email has. For removing a user, it holds email + one role that email had (it may have had others).
- Return the new state of the email + roles in the workspace. That would be inconsistent with other delete commands, but would mean we're using `UFWorkspaceUser` in a consistent way.
- Return just the role that got removed. This seemed useless because the user supplied the role in the command, so they don't need the cli to spit it back out.

Instead of trying to pick a less confusing option, I decided to just remove the json output and let users call `list-users` with JSON format to get the current state of the user they're interested in.

Reply:
Makes sense. Either returning the new state of email + roles or nothing both sound good to me.

Reply:
Thanks, returning nothing makes sense to me due to potential confusion and the ability to infer the resulting state.